pkg/util/azureclient/storage/storage.go

```diff
@@ -93,3 +101,28 @@ type blob struct {
 }
 
 var _ Blob = &blob{}
+
+func (b *blob) CreateBlockBlobFromReader(blob io.Reader, options *storage.PutBlobOptions) error {
+	ctx := context.Background()
+	data, err := ioutil.ReadAll(blob)
+	if err != nil {
+		return err
+	}
+	sr := io.NewSectionReader(bytes.NewReader(data), 0, int64(len(data)))
```

Comment:
can just use bytes.NewReader within the anonymous function below?

Reply:
i'll try